### DISCUSSION TITLE
Successful deployment - but cannot see changes on my site

### DISCUSSION THREAD

**Discussion by sutcliffe121** [deployphp/deployer#2872] [Help needed] [answered]
- Deployer version: deployer:v7.0.0-rc.3
- Deployment OS: Windows 10 Pro 64-bit
- Laravel 8.33.1
- nginx ubuntu server, built using Larasail

My deployment was successful, yet when I go to my site, I cannot see the changes I committed.

I suspect the issue might be something to do with symlinks or the fpm reload but I am not sure what steps to take. I am using digital ocean droplet with a nginx larasail build.

Any suggestions?

I have read (https://deployer.org/docs/7.x/avoid-php-fpm-reloading). When I add 'echo $_SERVER['SCRIPT_FILENAME'];' to a page, it does not include 'current'. 

```
λ php deployer.phar deploy
task deploy:info
[prod] info deploying master
task deploy:setup
task deploy:lock
task deploy:release
task deploy:update_code
task deploy:shared
task deploy:writable
task deploy:vendors
task artisan:storage:link
task artisan:view:cache
task artisan:config:cache
task deploy:symlink
task php-fpm:reload
[prod] warning Avoid reloading php-fpm [ï.at/avoid-php-fpm-reloading]
task deploy:unlock
task deploy:cleanup
task deploy:success
[prod] info successfully deployed!
```

**Comment by sutcliffe121** [comment, marked as answer]
Found the problem, silly mistake.

I hadn't set the nginx root to look at the 'current' folder. Doh!

All sorted now.

**Reply by antonmedv** [reply]
BTW, check out new provision recipe. It can properly configure an empty host.
